### ISSUE TITLE
MS2

### ISSUE THREAD

**Issue by flibbi2** [sarnau/Inside-The-Loxone-Miniserver#35]
Are there plans to do this work with Miniserver Gen2?

**Comment by sarnau** [comment]
No, I removed my Miniserver from the electrical panel – it is no longer needed and the rest of the house is KNX and Modbus. I removed all proprietary stuff.
